--- a/Makefile
+++ b/Makefile
@@ -677,27 +677,23 @@ raudio.o : raudio.c raylib.h
 # Compile raygui module
 # NOTE: raygui header should be distributed with raylib.h
 raygui.o : raygui.c
-	$(CC) -c $< $(CFLAGS) $(INCLUDE_PATHS)
+	$(CC) -c $< $(CFLAGS) $(INCLUDE_PATHS) -I$(RAYLIB_MODULE_RAYGUI_PATH)
 raygui.c:
 ifeq ($(PLATFORM_SHELL), cmd)
-	@echo #define RAYGUI_IMPLEMENTATION > raygui.c
-	@echo #include "$(RAYLIB_MODULE_RAYGUI_PATH)/raygui.h" >> raygui.c
+	@echo '#include "raygui.h"' > raygui.c
 else
-	@echo "#define RAYGUI_IMPLEMENTATION" > raygui.c
-	@echo "#include \"$(RAYLIB_MODULE_RAYGUI_PATH)/raygui.h\"" >> raygui.c
+	@echo "#include \"raygui.h\"" > raygui.c
 endif
 
 # Compile physac module
 # NOTE: physac header should be distributed with raylib.h
 physac.o : physac.c
-	$(CC) -c $< $(CFLAGS) $(INCLUDE_PATHS)
+	$(CC) -c $< $(CFLAGS) $(INCLUDE_PATHS) -I$(RAYLIB_MODULE_PHYSAC_PATH)
 physac.c:
 ifeq ($(PLATFORM_SHELL), cmd)
-	@echo #define PHYSAC_IMPLEMENTATION > physac.c
-	@echo #include "$(RAYLIB_MODULE_PHYSAC_PATH)/physac.h" >> physac.c
+	@echo '#include "physac.h"' >> physac.c
 else
-	@echo "#define PHYSAC_IMPLEMENTATION" > physac.c
-	@echo "#include \"$(RAYLIB_MODULE_PHYSAC_PATH)/physac.h\"" >> physac.c
+	@echo "#include \"physac.h\"" >> physac.c
 endif
 # Compile android_native_app_glue module
 android_native_app_glue.o : $(NATIVE_APP_GLUE)/android_native_app_glue.c
@@ -745,6 +741,8 @@ ifeq ($(ROOT),root)
 		cp --update raylib.h $(RAYLIB_H_INSTALL_PATH)/raylib.h
 		cp --update raymath.h $(RAYLIB_H_INSTALL_PATH)/raymath.h
 		cp --update rlgl.h $(RAYLIB_H_INSTALL_PATH)/rlgl.h
+		cp --update $(RAYLIB_MODULE_PHYSAC_PATH)/physac.h $(RAYLIB_H_INSTALL_PATH)/physac.h
+		cp --update $(RAYLIB_MODULE_RAYGUI_PATH)/raygui.h $(RAYLIB_H_INSTALL_PATH)/raygui.h
 		@echo "raylib development files installed/updated!"
     else
 		@echo "This function currently works on GNU/Linux systems. Add yours today (^;"
@@ -772,6 +770,8 @@ ifeq ($(ROOT),root)
 		rm --force --interactive --verbose $(RAYLIB_H_INSTALL_PATH)/raylib.h
 		rm --force --interactive --verbose $(RAYLIB_H_INSTALL_PATH)/raymath.h
 		rm --force --interactive --verbose $(RAYLIB_H_INSTALL_PATH)/rlgl.h
+		rm --force --interactive --verbose $(RAYLIB_H_INSTALL_PATH)/physac.h
+		rm --force --interactive --verbose $(RAYLIB_H_INSTALL_PATH)/raygui.h
 		@echo "raylib development files removed!"
         else
 		@echo "This function currently works on GNU/Linux systems. Add yours today (^;"
