--- a/src/Makefile
+++ b/src/Makefile
@@ -512,9 +512,11 @@ ifeq ($(PLATFORM),PLATFORM_DESKTOP)
         LDLIBS = -lGL -lpthread
     endif
     ifeq ($(USE_EXTERNAL_GLFW),TRUE)
-        # Check the version name. If GLFW3 was built manually, it may have produced
-        # a static library known as libglfw3.a. In that case, the name should be -lglfw3
-        LDLIBS = -lglfw
+    ifeq ($(OS),Windows_NT)
+        LDLIBS += -L$(libdir) -lglfw3
+	else
+		LDLIBS += -lglfw
+	endif
     endif
 endif
 ifeq ($(PLATFORM),PLATFORM_RPI)
